server/data/assessmentStateService.ts

```diff
@@ -223,7 +224,14 @@ export class AssessmentStateService {
   }
 
   async getWorkingAssessmentVersion(stateKey: StateKey) {
-    return (await this.store.getWorkingAssessment(stateKey)).assessment.version
+    try {
+      return (await this.store.getWorkingAssessment(stateKey)).assessment.version
+    } catch (err) {
+      logger.info(
+        `Unable to fetch version for User Id : ${stateKey.userId} and AssessmentType : ${stateKey.assessmentType}`,
+      )
+      return 1
```

Comment:
The ticket says remove version defaults, isn't this effectively a default to 1 when it errors?

Reply:
Done the changes to throw error